### PR TITLE
chore: provision

### PR DESCRIPTION
**Pull request by gcedo**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,102 +2,111 @@ import Button from '@economist/component-link-button';
...
         </div>
-      </nav>
-    );
-  }
+      </div>
+    </nav>
+  );
 }
+
+SectionsCard.propTypes = {
```
Comment (keithamus) [inline src/index.js]: Could you wrap this in `if(process.env.NODE_ENV !== 'production')` please. We have this on some others...

Comment (fabiosantoscode) [inline src/index.js]: I wouldn't do that. NODE_ENV is production in stage as well.

Comment (gcedo) [inline src/index.js]: :+1:

Comment (keithamus) [inline src/index.js]: @fabiosantoscode that's fine. Stage is meant to look like production. PropTypes are only a development feature and so we should have error free code (at least for proptypes) by the time it hits stage.

**Comment by keithamus** [comment]
Needs rebase @gcedo :smile:

**Comment by gcedo** [comment]
This PR was provisioning with 1.5.x, master is provisioned with 1.7.x, closing.

**Pull request closed without merging**